check/mode-lowmem.c

```diff
@@ -2629,6 +2630,12 @@ static int check_inode_item(struct btrfs_root *root, struct btrfs_path *path)
 		return err;
 	}
 
+	if (inode_id == BTRFS_FREE_INO_OBJECTID) {
+		error("subvolume %lld has deprecated inode cache",
```

Comment:
If this is an error then it should also exit at some point. As it is now it looks more like a warning.

Reply:
I can change that to warning, but even if we hit an error, as long as it's not critical, we should always continue other than exiting.
Or we will not fully check the whole filesystem.